Formula/libmarpa.rb

```diff
@@ -19,15 +19,15 @@ class Libmarpa < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "cmake" => :build
+  depends_on "emacs" => :build
   depends_on "libtool" => :build
+  depends_on "texinfo" => :build
   depends_on "texlive" => :build
 
-  uses_from_macos "texinfo" => :build
-
   def install
     ENV.deparallelize
     inreplace "work/etc/libmarpa.pc.in", "prefix=\".\"", "prefix=\"#{prefix}\"" if build.head?
-    system "make", "dists"
+    system "make", "dist"
```

Comment:
I think this change is causing failure on stable tarball. The change is not yet released (https://github.com/jeffreykegler/libmarpa/commit/275a2c43b6c7cfefa6ea47945fda1cb6399076af) so it will need to be a different Make target on stable and head builds.

Reply:
You're right!  Sorry, will fix.

Reply:
Done